website/content/en/docs/building-operators/golang/quickstart.md

```diff
@@ -34,7 +34,7 @@ Create a simple Memcached API:
 operator-sdk create api --group cache --version v1 --kind Memcached --resource=true --controller=true
 ```
 
-### Configuring your test environment
+### You must configure your test environment
```

Comment:
Configure Test Environment (REQUIRED for building the operator)

Reply:
I disagree with changing the headers. Can we add a sentence in this section (and possibly in the section that includes `make docker-build`) about the reasons for the necessity of this step.

Reply:
let us know wdyt now @joelanford ?